packages/helpers-gatsby/src/Tags.js

```diff
@@ -36,37 +36,54 @@ const Tags = (props) => {
         visibility: props.visibility,
     }
 
+    let keyIndex = 0
+    const generateKey = (pre) => {
+        keyIndex = keyIndex + 1
+        return `${pre}_${keyIndex}`
+    }
+
     if (props.separator) {
-        opts.separator = React.isValidElement(props.separator) ? props.separator :
-            <><span className={props.separatorClasses}>{props.separator}</span></>
+        if (React.isValidElement(props.separator)) {
+            opts.separator = props.separator
+        } else {
+            Object.defineProperty(opts, 'separator', {
+                get() {
+                    return (
+                        <span className={props.separatorClasses} key={generateKey('separator')}>
+                            {props.separator}
+                        </span>
+                    )
+                },
+            })
+        }
     }
 
     if (props.prefix) {
         opts.prefix = React.isValidElement(props.prefix) ? props.prefix :
-            <><span className={props.prefixClasses}>{props.prefix}</span></>
+            <span className={props.prefixClasses} key="prefix">{props.prefix}</span>
```

Comment:
Yes, because this will only be rendered once per "call", it only need this hardcoded `key` to satisfy Gatsby